modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -86,6 +84,17 @@ s| 4.0.0 | Strict
 s| 4.1.0  and later | Flexible
 |===
 
+== Troubleshooting Gatekeeper Issues in Mule v3.x
+
+==== Gatekeeper does not unblock the API when a resource level policy is applied to it 
+
+In Mule 3, you can only apply resource-level policies to a RAML proxy or to an API implementation containing an APIKit router. Gatekeeper might repeatedly block an API when a resource-level policy is applied to it due to two main reasons:
+
+* The selected expressions for URI and method when configuring the resource-level policy do not match with any resource specified in the RAML definition. To avoid this type of error, use the `Preview Resource Matching` option in API Manager when configuring the specific resource-level policy. 
```

Comment:
Please use proper bullet-list format:
```suggestion
* The expressions for URI and method selected when configuring the resource-level policy do not match any resource specified in the RAML definition. 
+
To avoid this type of error, use the `Preview Resource Matching` option in API Manager when configuring the specific resource-level policy. 
```

Reply:
Done! Thanks!